common/include/test_assert.h

```diff
@@ -28,25 +28,39 @@
 #ifdef try_status
 #undef try_status
 #endif
+#if defined(EXPECT_EQ)
```

Comment:
This is a good transitional strategy but longer term we should refactor tests to use `EXPECT_EQ` directly as we do want to migrate all tests to GTest.